packages/app/src/hooks/endpoint.ts

```diff
@@ -922,3 +923,30 @@ export const useEndpoint = (): UseEndpointReturn => {
   );
   return ret;
 };
+
+export const useEndpointGroupToggle = (id: string) => {
+  const [endpointGroups, setEndpointGroups] = useEndpointGroupToggleState();
+
+  const endpointGroup = endpointGroups.find((group) => group.id === id);
+  const isOpenToggle = endpointGroup
```

Comment:
Perhaps using optional chaining will improve readability, instead of using the ternary operator.